### ISSUE TITLE
Use Case that blocks login with unfriendly & undocumented fix

### ISSUE THREAD

**Issue by spersico** [appwrite/sdk-for-cli#32]
So:
I just started playing with appwrite.
- Installed appwrite in my local docker. Everything broke (makes sense, my local docker installation is super broken). Decided to go for the 1-click gitpod one. Created a project and a user there. Everything works!
- Installed the appwrite-cli package, and tried loggin-in.
I probably copied the wrong url or something like that because it threw the _Error Not Found_ (the _general_route_not_found_ one, if I put the `--verbose` option).
**Thing is, I couldn't change the URL to fix my issue!** On **every** login attempt, the CLI seems to cache the URL, so I got myself in a loop error situation.

Because I was pretty sure that the problem was local, not in my Gitpod instance, I tried clearing the cache somehow:
- Tried uninstalling the npm package.
- Tried the `appwrite logout`
- Finally, I gave up, and started looking up the code for appwrite-cli, and I guessed (correctly) that this is where the data is stored: https://github.com/appwrite/sdk-for-cli/blob/4ef735949296dd5eea6bd9620bac93977f450182/lib/config.js#L202

So, I just deleted that and got to try again until I figured out my error (I was hitting the appwrite root URL instead of the appwrite project server's URL).
So the fix was finally:
```bash
  rm -rf ~/.appwrite/prefs.json
  appwrite login
```
 ...and then try again

---------------
My point? Needing to fix issues by reading the code, it's not really user-friendly. **The config location should be disclosed in a more visible place, IMHO.**
**But more importantly, I believe that after a login error (and specially after the Not Found error), the URL shouldn't be cached. Otherwise, the error state is perpetuated. And I'm uncertain if ever caching that URL is a good idea because changing the appwrite destination gets more cumbersome that it needs to, doesn't it?**

**Comment by christyjacob4** [comment]
@spersico that's great feedback, but did you get a chance to read this section of the docs ? 
https://appwrite.io/docs/command-line#configuration

It talks about how you can change any of the CLI configuration parameters. 

So the feature exists.. You don't need to delete any files to re-configure your endpoint. 

I think it boils down to communication... How do you suggest we could communicate this better ? 

Regarding the "Do not cache the endpoint" if it throws an error - I think thats a good suggestion and we could potentially fix it.

**Comment by spersico** [comment]
Hey there!  👋
I took a look at the URL and you are correct. I didn't see that one.
Regarding the communication thing... I can think of 2 things, based of what I did to figure out the issue:
- I googled the error code, and then the specific error code (the verbose version).
- I tried to search the login help page of the CLI
- I searched "change endpoint appwrite login" or something like that.

So, the way to approach the communication side of things would be:
- The [errors page](https://appwrite.io/docs/error-codes), could provide hints of where to look to solve the problem, like you already do for error codes `429` and `500`.
- The other one, is to add a parameter/option to the `login` command to allow the CLI user to set the endpoint using the same command that sets the endpoint, to do everything at once (and add that information to the `login help` command.
It could end up looking like appwrite login -endpoint="http://www.google.com/v1", or something like that.
- Another option is to add the information to the `login help` command, to indicate the user how to change the endpoint if needed.
